src/app/interfaces/strongRoute.ts

```diff
@@ -113,22 +116,36 @@ export class StrongRoute {
    */
   compileRoutes(callback: RouteConfigCallback): Routes {
     const rootRoute = this.root;
+
+    const sortRoutes = (a: Route, b: Route): -1 | 0 | 1 => {
+      // If one of the routes is a parameter route
+      if (a.path.charAt(0) === ":" || b.path.startsWith(":")) {
+        // If both are parameter routes, they are equal
+        if (a.path.charAt(0) === ":" && b.path.startsWith(":")) {
+          return 0;
+        }
+        // Else give priority to the non-parameter route
+        return a.path.charAt(0) === ":" ? 1 : -1;
```

Comment:
you pull the first char out of each string so many times, might as well just

```
aParamRoute = a.path.charAt(0) === ":";
bParamRoute = b.path.charAt(0) === ":";
```
at the start of the method